src/libs/OptionsListUtils.js

```diff
@@ -491,6 +492,10 @@ function getOptions(reports, personalDetails, {
         const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(report);
         const logins = report.participants || [];
 
+        if (isPolicyExpenseChat && report.isOwnPolicyExpenseChat && !includeOwnedWorkspaceChats) {
+            return;
+        }
+
```

Comment:
I know this is pretty self explanitory, but just to make it slightly faster to understand, maybe we could add a comment explaining this logic.

Reply:
I am on the same boat, as said in another comment I think any of this logic which is more tied with our internal jargon is great to explain in plain english too

Reply:
I always welcome a suggestion. I'm usually pretty cautious about adding too much context if the code is self-explanatory. If people need a deeper understanding about how the product works they can always ask us.

Like I started writing this comment...

>Most lists will not display the user's owned Policy Expense Chats (a.k.a. Workspace Chats). These chats are between a workspace member and their workspace admins and used by the workspace member to request money.

But do we need a comment like this next to every mention of "policy expense chat" ?

Reply:
Anyways, let me know what specifically you guys think needs to be here and I will update it.

Reply:
@marcaaron thanks for context. I am happy to leave this one like its now given your comment here explains the property well https://github.com/Expensify/App/pull/16967/files#r1159042123

Reply:
The other comment I thought of was...

>If we have a workspace chat and it is the user's own policy expense chat and we don't want to include them then return early

But doesn't strike me as super useful because it just describes what the code does.

Reply:
For me personally its just the question of Why is is shown for request but not for send or split which might not be clear for external people. I think what we have there in the param comment is good. Thank you!

Reply:
Yeah, I think we can probably do without the comment after all 👍